server/game/cards/01-Core/VerteranBuilder.js

```diff
@@ -1,32 +1,21 @@
 const DrawCard = require('../../drawcard.js');
 
 class VerteranBuilder extends DrawCard {
-    setupCardAbilities() {
+    setupCardAbilities(ability) {
         this.action({
-            title: 'Sacrifice this card to stand a location',
-            method: 'sacrifice'
+            title: 'Stand a location',
+            cost: ability.costs.sacrificeSelf(),
+            target: {
+                activePromptTitle: 'Select a location',
+                cardCondition: card => card.location === 'play area' && card.controller === this.controller && card.getType() === 'location'
```

Comment:
same here: Veteran Builder can be used to stand opponents' locations, so we should remove the controller check